project-base/storefront/utils/user/useUserMenuItems.ts

```diff
@@ -0,0 +1,46 @@
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import useTranslation from 'next-translate/useTranslation';
+import { useComparison } from 'utils/productLists/comparison/useComparison';
+import { useWishlist } from 'utils/productLists/wishlist/useWishlist';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+
+type UserMenuItemType = {
+    link: string;
+    text: string;
+    count?: number;
+};
+
+export const useUserMenuItems = (): UserMenuItemType[] => {
+    const { t } = useTranslation();
+    const { url } = useDomainConfig();
+    const { comparison } = useComparison();
+    const { wishlist } = useWishlist();
+    const [customerOrdersUrl, customerEditProfileUrl, productComparisonUrl, wishlistUrl] =
+        getInternationalizedStaticUrls(
+            ['/customer/orders', '/customer/edit-profile', '/product-comparison', '/wishlist'],
+            url,
+        );
+
+    const userMenuItems: UserMenuItemType[] = [
+        {
+            text: t('Account settings'),
```

Comment:
In header, this still says `Edit profile`. It might be confusing to use 2 different texts for the same link.